solidity/contracts/converter/types/liquidity-pool-v1/LiquidityPoolV1Converter.sol

```diff
@@ -297,6 +297,20 @@ contract LiquidityPoolV1Converter is LiquidityPoolConverter {
         removeLiquidityFromPool(reserveTokens, reserveMinReturnAmounts, totalSupply, _amount);
     }
 
+    /**
+      * @dev calculates the amount of each reserve token entitled for a given amount of pool tokens

```

Comment:
`calculates` is an implementation detail, can just put `returns` there

Reply:
It appears in a few other places. Let's fix that once we merge both PRs into 0.6.17, because of this entanglement (of handling two PRs).

Reply:
Done